ext/opcache/ZendAccelerator.c

```diff
@@ -5050,11 +5038,6 @@ static zend_result accel_finish_startup(void)
 		return SUCCESS;
 	}
 
-#ifdef ZEND_WIN32
-	zend_accel_error_noreturn(ACCEL_LOG_ERROR, "Preloading is not supported on Windows");
-	return FAILURE;
-#else /* ZEND_WIN32 */
-
```

Comment:
Shouldn't you check for a "good" SAPI and/or disable "attaching" for "bad" ones?